.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
```

Comment:
You should remove all .idea folders from the repo (and add them to .gitignore). That applies for everything under .idea/ and everything under /server/.idea/